docs/introduction/di_in_python.rst

```diff
@@ -49,7 +49,7 @@ Coupling and cohesion are about how tough the components are tied.
 - **High cohesion**. High cohesion is like using the screws. Very easy to disassemble and
   assemble back or assemble a different way. It is an opposite to high coupling.
 
-When the cohesion is high the coupling is low.
+High cohesion often correlates with loose coupling, and vice versa.
```

Comment:
What do you think about something like that:

“Cohesion often correlates with coupling. Higher cohesion usually leads to lower coupling, and vice versa.”

Reply:
LGTM! Changed in f8ec061